spark/v2.4/spark/src/main/java/org/apache/iceberg/spark/actions/BaseSparkActions.java

```diff
@@ -44,6 +44,11 @@ public DeleteOrphanFiles deleteOrphanFiles(Table table) {
     return new BaseDeleteOrphanFilesSparkAction(spark, table);
   }
 
+  @Override
+  public RewriteDataFiles rewriteDataFiles(Table table) {
+    return new BaseRewriteDataFilesSparkAction(spark, table);
```

Comment:
oh, it is an abstract class!

Reply:
@RussellSpitzer : I should have something like Spark3BinPackStrategy and Spark3SortStrategy.
Then call that from BaseRewriteDataFilesSparkAction2 ?

Reply:
But we don't have `SparkDistributionAndOrderingUtil`.

I am confused now. We need to bring back deprecated action for this or block rewrite data files for spark 2.4 or fix it by bringing SparkDistributionAndOrderingUtil and other classes form spark3 [not sure if it works]